res/database/Default/user/jorgaeff.xml

```diff
@@ -200,9 +232,11 @@
 			<availability script="TIME_YEAR=1985" />
 			<title>
 				<de>Michailow wird KPdvSS-Generalsekretär</de>
+				<en>Michailow becomes KPdvSS Secretary General</en>
 			</title>
 			<description>
 				<de>Im Alter von 54 Jahren wird Sergei Michailow zum jüngsten Generalsekretär in der Geschichte der Kommunistischen Partei der vereinigten Sowjet-Staaten (KPdvSS) gewählt.</de>
+				<en>At the age of 54, Sergei Mikhailov is elected the youngest General Secretary in the history of the Communist Party of the United Soviet States (CPdvSS).</en>
```

Comment:
Consistency CPd/KPd in title/description (and the remaining news entries).

Reply:
ah, that was botched up either way. CPSU is the correct English notation. Fixing.